collection-scripts/gather

```diff
@@ -74,13 +74,12 @@ case "$CLUSTER" in
     oc adm inspect  ns/hive  --dest-dir=must-gather
     oc adm inspect  multiclusterengines.multicluster.openshift.io --all-namespaces  --dest-dir=must-gather
     oc adm inspect hiveconfigs.hive.openshift.io --all-namespaces  --dest-dir=must-gather
-    
+
     oc adm inspect clusterserviceversions.operators.coreos.com --all-namespaces  --dest-dir=must-gather
     oc adm inspect subscriptions.operators.coreos.com      --all-namespaces  --dest-dir=must-gather
     oc adm inspect installplans.operators.coreos.com --all-namespaces  --dest-dir=must-gather
     oc adm inspect operatorgroups.operators.coreos.com --all-namespaces  --dest-dir=must-gather
 
-    oc adm inspect baremetalassets.inventory.open-cluster-management.io --all-namespaces  --dest-dir=must-gather
```

Comment:
@gurnben This is the only change, others are just auto-save format, I can revert them if needed.